package.json

```diff
@@ -13,7 +13,8 @@
     "reflect-metadata": "^0.1.3",
     "rxjs": "5.0.0-beta.6",
     "zone.js": "^0.6.12",
-    "moment": "2.14.1"
+    "moment": "2.14.1",
+    "ng2-translate": "^2.2.2"
```

Comment:
Please do not use `^` on any packages which work with ionic2 or angular2. Since they are still in beta, future versions of this package could break.